File: examples/javascript/basic.js
examples/javascript/basic.js
const API_KEY = "dg_live_..." // replace with your key

async function validateToolCall() {
  const res = await fetch("https://driftgentic.com/functions/v1/validate", {
    method: "POST",
    headers: {
      "Authorization": `Bearer ${API_KEY}`,
      "Content-Type": "application/json"
    },
    body: JSON.stringify({
      tool: "stripe/create_customer",
      payload: {
        email: "test@example.com"
      }
    })
  })

  const result = await res.json()

  if (!result.valid) {
    console.error("Validation failed:", result.errors)
  } else {
    console.log("Validation

File: examples/javascript/examples/python/examples/python/basic.py
import requests

API_KEY = "dg_live_..."  # replace with your key

res = requests.post(
    "https://driftgentic.com/functions/v1/validate",
    headers={
        "Authorization": f"Bearer {API_KEY}",
        "Content-Type": "application/json"
    },
    json={
        "tool": "stripe/create_customer",
        "payload": {
            "email": "test@example.com"
        }
    }
)

result = res.json()

if not result.get("valid"):
    print("Validation failed:", result.get("errors"))
else:
    print("Validation passed")
